eng/targets/Settings.props

```diff
@@ -11,6 +11,9 @@
 
     <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
 
+    <!-- Disable the EOL check to ensure no warnings in the insertion runs -->
```

Comment:
No warnings about what framework? Can we disable this more specifically?

Reply:
I did this differently in my preview 7 PR. https://github.com/dotnet/roslyn/pull/80020/files#diff-6b588e7e172476b7649e5ae02e03d25b9b8f974ea2ec0324a5cea7272a077af4R25-R26

Reply:
Can we just disable in the specific projects?